src/index.html

```diff
@@ -85,6 +85,7 @@
             <calcite-button class="open" value="two">Open popover two</calcite-button>
             <calcite-button class="open" value="three">Open popover three</calcite-button>
           </p>
+          <instant-apps-export id="export"></instant-apps-export>
```

Comment:
Not for this PR - I think we should create a gallery of test apps for each component as this library is starting to grow, and the root index.html will be a page which displays an gallery item for each test app